micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSEdit.js

```
@@ -37,79 +37,73 @@ const MDMSEdit = ({...props}) => {
     },
     config: {
       enabled: moduleName && masterName && true,
-      select: (data) => { 
-        const uniqueFields = data?.SchemaDefinitions?.[0]?.definition?.["x-unique"]
-        const updatesToUiSchema = {}
-        uniqueFields.forEach(field => updatesToUiSchema[field] = {"ui:readonly":true})
-        return {schema:data?.SchemaDefinitions?.[0],updatesToUiSchema}
+      select: (data) => {
+        const uniqueFields = data?.SchemaDefinitions?.[0]?.definition?.["x-unique"];
+        const updatesToUiSchema = {};
+        uniqueFields.forEach((field) => (updatesToUiSchema[field] = { "ui:readonly": true }));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)